README.md

```diff
@@ -94,26 +94,30 @@ EXAMPLE
 
 _See code: [src/commands/destroy-backend.ts](https://github.com/dgraph-io/slash-graphql-cli/blob/v1.10.0/src/commands/destroy-backend.ts)_
 
-## `slash-graphql drop-data`
+## `slash-graphql drop`
 
-Drop all data in your backend
+Drop data, schema or unused types/fields in your backend.
 
 ```
 USAGE
-  $ slash-graphql drop-data
+  $ slash-graphql drop
```

Comment:
this example doesn't work. Could you add 4-5 examples, one for each thing you can drop